english/ghostbusters/ghostbusters.md

```diff
@@ -44,7 +44,7 @@ and sparkle as your body and mind are sent reeling.
 
 You awaken to bluriness and a numbing headache as you wonder
 
-WHERE AM I ???
+WHERE AM I ??? You sense a cloaked figure with electric [blue eyes] (../goosebumps/goosebumps.md) appraoch you from the side.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.